tests/test_form.py

```diff
@@ -235,9 +235,9 @@ def test_set_select_multiple(options):
     # When a browser submits multiple selections, the qsl looks like:
     #  name=option1&name=option2
     if not isinstance(options, list) and not isinstance(options, tuple):
-        expected = (('instrument', options),)
+        expected = [('instrument', options)]
```

Comment:
I agree that we need to test the sequence, not the set, but changing from tuple (`(...)`) to list (`[...]`) seems unnecessary (but not really harmful either, just that I like immutable things when I don't need to mutate them).

Reply:
I agree that would probably be preferable, but since the output of `parse_qsl` is explicitly a list object, I saw only two sensible options:
1. Define the expected output as a list.
2. Convert the expected output to a list in `mock_post`.

I chose the first option since it seemed like the most direct comparison and it could not hide any errors behind a cast to another type. However, I'm open to alternatives if you see something better.

Reply:
It makes sense, indeed.